src/nvim/lua/vim.lua

```diff
@@ -1,2 +1,62 @@
 -- TODO(ZyX-I): Create compatibility layer.
-return {}
+--{{{1 package.path updater function
+-- Last inserted paths. Used to clear out items from package.[c]path when they
+-- are no longer in &runtimepath.
+local last_nvim_paths = {}
+local function _update_package_paths()
+  local new_last_nvim_paths = {}
```

Comment:
I get that this is meant to be assigned to `last_nvim_paths` at the end of this function, but `new_last_` sounds a bit strange. How about just `new_nvim_paths`?